docs/UserGuide.md

```diff
@@ -51,14 +51,8 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
   e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
 
-* Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
-
-* Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
-
 * Parameters can be in any order.<br>
-  e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
+  e.g. if the command specifies `n/NAME i/IC_NUMBER`, `i/IC_NUMBER n/NAME` is also acceptable.
```

Comment:
Our feature spec uses the keyword 'IC' instead of 'IC_NUMBER'
![image](https://github.com/AY2324S2-CS2103T-F14-2/tp/assets/110999129/50ecf506-8a97-4564-ad8e-9b73c14981eb)

We should probably stick to one or the other when writing. I think IC_NUBER is better so maybe we can change the feature spec to say IC_NUMBER?

Reply:
Fixed in #27